pages/doc.js

```diff
@@ -1,21 +1,20 @@
 /* global docsearch:readonly */
 
-import React, { Component } from 'react'
+import React, { useCallback, useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import Error from 'next/error'
+import Router from 'next/router'
 // components
 import Page from '../src/Page'
 import { HeadInjector } from '../src/Documentation/HeadInjector'
 import Hamburger from '../src/Hamburger'
 import SearchForm from '../src/SearchForm'
 import SidebarMenu from '../src/Documentation/SidebarMenu/SidebarMenu'
-import Loader from '../src/Loader/Loader'
-import Page404 from '../src/Page404'
 import Markdown from '../src/Documentation/Markdown/Markdown'
 import RightPanel from '../src/Documentation/RightPanel/RightPanel'
 // utils
 import fetch from 'isomorphic-fetch'
 import kebabCase from 'lodash.kebabcase'
-// constants
-import { HEADER } from '../src/consts'
```

Comment:
is it used anywhere else? remove?

Reply:
From that I see it is used in the `TopMenu` and in the `Tooltip` components.

Reply:
👍